nosepipe.py

```diff
@@ -147,12 +147,16 @@ def __call__(self, result):
             useshell = True
 
         self.logger.debug("Executing %s", " ".join(argv))
-        popen = subprocess.Popen(argv,
-                                 cwd=self._cwd,
-                                 stdout=subprocess.PIPE,
-                                 stderr=subprocess.STDOUT,
-                                 shell=useshell,
-                                 )
+        try:
+            popen = subprocess.Popen(argv,
+                                     cwd=self._cwd,
+                                     stdout=subprocess.PIPE,
+                                     stderr=subprocess.STDOUT,
+                                     shell=useshell,
+                                     )
+        except OSError, e:
```

Comment:
Just tried this with Python 3 and noticed this is Python 2 only code - it should instead be `except OSError as e:`.